webapp/interop_handler.go

```diff
@@ -7,15 +7,26 @@ package webapp
 import (
 	"net/http"
 
+	"github.com/gorilla/mux"
 	"github.com/web-platform-tests/wpt.fyi/shared"
 )
 
-type interop2022Data struct {
+type interopData struct {
 	Embedded bool
+	Year     string
 }
 
-// interop2022Handler handles GET requests to /interop-2022
-func interop2022Handler(w http.ResponseWriter, r *http.Request) {
+// interopHandler handles GET requests to /interop
+func interopHandler(w http.ResponseWriter, r *http.Request) {
+	name := mux.Vars(r)["name"]
+	year := mux.Vars(r)["year"]
```

Comment:
We should do a year check here. e.g. interop-0 returns:

Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'focus_areas')
    at new InteropDataManager (interop.js:24:32)
    at InteropDashboard.ready (interop.js:604:24)
    at InteropDashboard._enableProperties (properties-changed.js:362:14)
    at InteropDashboard.connectedCallback (properties-mixin.js:222:11)
    at InteropDashboard.connectedCallback (element-mixin.js:685:13)
    at interop.js:748:23

Reply:
Added a fix for this which should redirect to `interop-2022` now for invalid years.